package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "0.9.2",
+  "version": "0.9.2-d1",
```

versions shouldn't be bumped in PRs.
```suggestion
  "version": "0.9.2",
```

---

Sorry for tampering the branch. Clean now